liip/LiipTestFixturesBundle
# [Feature]: Add docker setup and an updated README for people who want to contribute

# Feature: Docker Setup for Testing

## Motivation

I recently opened an issue (https://github.com/liip/LiipTestFixturesBundle/issues/259) and want to contribute to the project to help improve it. However, as a newcomer to Symfony and this specific project, I faced challenges getting started.

Firstly, I couldn't find a README or any documentation to guide me through the project setup. Additionally, running tests or performing other tasks seemed reliant on having PHP, Composer, MySQL, and other services installed locally, which is not an ideal practice.

To address this, I've created this branch to enhance the onboarding process for new contributors like myself.

## How to Test

1. Clone this branch.
2. Run: `docker-compose up -d` to install the necessary services.
3. Enter the container: `docker-compose exec php-fpm bash`.
4. Install dependencies: `composer install`.
5. Run tests and ensure everything works as expected: `/vendor/bin/phpunit`.
